reactive/load_balancer.py

```diff
@@ -208,6 +213,7 @@ def upgrade_charm():
     if is_state("certificates.available") and is_state("website.available"):
         request_server_certificates()
     maybe_write_apilb_logrotate_config()
+    allow_lb_consumers_to_read_requests()
```

Comment:
I see this is called during the upgrade-charm hook, but what about new installs? I would consider setting it in a `@hook("install")` handler too, or maybe set the configuration immediately prior to accessing `all_requests`.

Reply:
yep, sorry, i hadn't checked in what i had successfully tested the day before.  The test failures now reflect the failure of the missing method in LBConsumers